code/__DEFINES/rust_g.dm

```diff
@@ -6,22 +6,22 @@
 #define RUSTG_JOB_ERROR "JOB PANICKED"
 
 #define rustg_dmi_strip_metadata(fname) call(RUST_G, "dmi_strip_metadata")(fname)
-#define rustg_dmi_create_png(fname,width,height,data) call(RUST_G, "dmi_create_png")(fname,width,height,data)
+#define rustg_dmi_create_png(path, width, height, data) call(RUST_G, "dmi_create_png")(path, width, height, data)
+
+#define rustg_noise_get_at_coordinates(seed, x, y) call(RUST_G, "noise_get_at_coordinates")(seed, x, y)
```

Comment:
Is this actually more efficient than just dumping whole data at once or it's some persistence/other requirement.

Reply:
I need to be able to get the seed of specific coordinates, in specific conditions. So getting it beforehand is not very feasible